cli/flags.rs

```diff
@@ -606,10 +606,9 @@ fn test_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
   let allow_none = matches.is_present("allow-none");
   let quiet = matches.is_present("quiet");
   let filter = matches.value_of("filter").map(String::from);
-  let coverage = matches.is_present("coverage");
 
-  if coverage {
-    flags.coverage = true;
+  if matches.is_present("coverage") {
```

Comment:
Default to a tempdir if no value is provided?

Reply:
Yes, please

Reply:
@lucacasonato objections? IIRC you were against a default.

Reply:
Yes, I dont think we should add a default value. How do you get the temp dir? You would need to pass it to `deno cover` to generate a report right?

Reply:
Yeah, it'd just move the "BREAKING" bit to the next PR really so marking as resolved.